docs/prometheus.md

```diff
@@ -0,0 +1,21 @@
+## syzkaller metrics
+
+syz-manager metrics are exposed at the URI : http://<syz-manager-http-uri>/metrics
+The prefix for metrics of importance is `syz_`
+
+These metrics are exported in prometheus format and can be ingested using following prometheus client configuration:
+```
+scrape_configs:
+- job_name: syzkaller
+  scrape_interval: 10s
+  static_configs:
+  - targets:
+    - localhost:56741
+```
+
+Values are reset to zero at syz-manager restart and only reflect for current execution of syz-manager.
+
+---
+
+References
+1. https://prometheus.io/docs/guides/go-application/
```

Comment:
The rest of this doc looks like it is intended for end users. This does not look to be important/useful for end users. Perhaps it's better placed in the source code.

Reply:
Agreed! Moving this to the code